dev-docs/design/how-lit-html-works.md

```diff
@@ -157,13 +157,15 @@ Evaluating a template expression is fast. It's only as expensive as the JavaScri
 
 Initiated by a call to lit-html's `render()` function, such as: `render(counterUi(0), container)` in the example.
 
-The `render()` function looks for an existing `_$litPart$` field on the container element from a previous call, or instantiates and assigns a new `ChildPart` on the container. Then, the rendering of the passed in `TemplateResult` is kicked off by calling `_$setValue` on the container's `ChildPart`.
+The `render()` function checks for a `_$litPart$` field on the container element, which will be a Lit `ChildPart` object if Lit has rendered into that container before. If it has, we can just tell that ChildPart to render the template by calling `_$setValue` on it, otherwise we'll instantiate a new ChildPart, assign it to the `_$litPart$` field on the container, and then call `_$setValue`. More on Parts, including ChildPart, below.
 
 ### 2.i. Prepare
 
-Source: [`_$getTemplate` method which contains and caches the **Prepare** phase](https://github.com/lit/lit/blob/5659f6eec2894f1534be1a367c8c93427d387a1a/packages/lit-html/src/lit-html.ts#L1562-L1568).
+Source: [the `_$getTemplate` method which contains and caches the **Prepare** phase](https://github.com/lit/lit/blob/5659f6eec2894f1534be1a367c8c93427d387a1a/packages/lit-html/src/lit-html.ts#L1562-L1568).
 
-When a unique `TemplateResult` is rendered for the very first time on the page it must be prepared. The end result of the preparation phase is an instance of the Lit `Template` class that contains a `<template>` element and list of metadata objects (called `TemplatePart`s) storing where the dynamic JavaScript values should be set on the DOM. This Lit `Template` object is cached by the `TemplateResult`'s unique `strings` template strings array so this phase is skipped if a `TemplateResult` from the same tag function call is ever encountered again.
+When a unique `TemplateResult` is rendered for the very first time on the page it must be prepared. The end result of the preparation phase is an instance of the Lit `Template` class that contains a `<template>` element and list of metadata objects (called `TemplatePart`s) storing where the dynamic JavaScript values should be set on the DOM. This Lit `Template` object is cached by the `TemplateResult`'s unique `strings` template strings array so this phase is skipped if a `TemplateResult` from the same tagged template literal is ever encountered again.
```

Comment:
Consider:

When a unique `TemplateResult` 

-> 

When a unique `html` tagged template literal is rendered


TemplateResults are always unique, they're keyed off the `strings` array

Reply:
Ah! great point! Thank you!